docker-run-opencv-ovms.sh

```diff
@@ -193,6 +193,7 @@ docker run --network host $cameras $TARGET_USB_DEVICE $TARGET_GPU_DEVICE --user
 -e RENDER_MODE=$RENDER_MODE $stream_density_mount \
 -e INPUTSRC_TYPE=$INPUTSRC_TYPE -e DISPLAY=$DISPLAY \
 -e cl_cache_dir=/home/pipeline-server/.cl-cache \
+-e RUN_PATH=`pwd` \
```

Comment:
would this be a problem if this is running from benchmark_scripts/ folder?

Reply:
I believe it would behave similar to DLStreamer and place the results in benchmark-scripts/results

Reply:
if this causes issue running from benchmark_scripts/ , we may need to use readlink to get the full absolute path to project's root results/ folder...